libs/application/templates/criminal-record/src/forms/CriminalRecordForm.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  buildForm,
+  buildDescriptionField,
+  buildMultiField,
+  buildSection,
+  buildSubmitField,
+  Form,
+  FormModes,
+  buildExternalDataProvider,
+  buildDataProviderItem,
+  buildCustomField,
+  DefaultEvents,
+} from '@island.is/application/core'
+import { m } from '../lib/messages'
+import { ApiActions } from '../shared'
+
+export const CriminalRecordForm: Form = buildForm({
+  id: 'CriminalRecordFormDraft',
+  title: '',
+  mode: FormModes.APPLYING,
+  children: [
+    buildSection({
+      id: 'externalData',
+      title: m.externalDataSection,
+      children: [
+        buildExternalDataProvider({
+          title: m.externalDataTitle,
+          id: 'approveExternalData',
+          subTitle: m.externalDataSubTitle,
+          checkboxLabel: m.externalDataAgreement,
+          dataProviders: [
+            buildDataProviderItem({
+              id: 'nationalRegistry',
+              type: 'NationalRegistryProvider',
+              title: m.nationalRegistryTitle,
+              subTitle: m.nationalRegistrySubTitle,
+            }),
+            buildDataProviderItem({
+              id: 'userProfile',
+              type: 'UserProfileProvider',
+              title: m.userProfileInformationTitle,
+              subTitle: m.userProfileInformationSubTitle,
+            }),
+            buildDataProviderItem({
+              id: 'payment',
+              type: 'FeeInfoProvider',
+              title: '',
```

Comment:
Intentionally no title here?

Reply:
Yes it's supposed to be hidden from the user because it contains no personal data.